git-pr-chain.py

```diff
@@ -268,7 +268,7 @@ def gh_branch_prefix():
         return git("config", "pr-chain.branch-prefix").strip()
     except subprocess.CalledProcessError:
         # git config exits with an error code if the config key is not found.
-        return ""
+        return gh_client().get_user().login + "/"
```

Comment:
I'm fine with this change in principle, but as-is this breaks existing users, right?  If I have some existing PRs and I don't have a branch prefix, then this will change the branch names of all my outstanding PRs?

Reply:
Yeah, you're right. It'll create new PRs for existing PR... In git-stacked-pr, we encode the full name in the commit message, including the PR link (once it's created), but git-pr-chain it's a separate config...

What do you think we can do to not breaking existing users' PR? I can separate this to its own PR if you want (so that the rest can go in)

Reply:
It's a little complex, but we could add a "time bomb"?  Basically the first time it runs it remembers the date (you could save it in the gitconfig) and prints a warning that at date + 1 month it will change the default?

Reply:
Sound like a good idea. I've removed this commit from the PR for now, an I can redo it, together with this change, in another PR.